lib/ImportFromC/HeaderToModel.cpp

```diff
@@ -1010,17 +1086,20 @@ bool DeclVisitor::handleUnionType(const clang::RecordDecl *RD) {
   uint64_t CurrentIndex = 0;
   for (const FieldDecl *Field : Definition->fields()) {
     if (Field->isInvalidDecl()) {
-      Errors.emplace_back("import-from-c: Invalid declaration for a union "
-                          "field.\n");
+      Errors.emplace_back("import-from-c failed: The declaration of the union "
+                          "field #'"
+                          + std::to_string(Union->Fields().size())
```

Comment:
`.size()`? :thinking:

Reply:
Can't you tell the field number?

Reply:
Aaah we're parsing, so this is fine because `size()` is also the index of the current field being parsed. nice.

Reply:
Yep, we're inserting them one at a time. If we go with indexing from 0, the number of already processed fields is our index.

Reply:
neat